openrtb_ext/bid_request_video.go

```diff
@@ -0,0 +1,268 @@
+package openrtb_ext
+
+import (
+	"github.com/mxmCherry/openrtb"
+)
+
+type BidRequestVideo struct {
+	// Attribute:
+	//   accountid
+	// Type:
+	//   string; required
+	// Description:
+	//   Unique ID of the stored request
```

Comment:
Fix above line!?

Reply:
I thought we ended saying this wasn't going to be an `accountid` but a sort of parent level config id?

Reply:
Yes. We use just the storedrequestid. We need to delete this.

Reply:
Deleted